app/src/main/java/ch/logixisland/anuto/entity/enemy/WeaponType.java

```diff
@@ -4,5 +4,6 @@
     None,
     Bullet,
     Laser,
-    Explosive
+    Explosive,
+    Glue
```

Comment:
Now that `Glue` is officially a `WeaponType` it would be nice if the `GlueEffect` class would check for the strong against `Glue` instead of ignoring `Flyer` instances specifically.